### PR TITLE
Camera and line improvements, small fixes

### PR DESCRIPTION
**Pull request by danirukun**
Add some more docs
Implement orthogonal line check
Add fixed capture device settings

### CODE REVIEW

File: src/VideoInput.cpp [hunk 1]
```diff
@@ -27,12 +28,24 @@ bool aic::VideoInput::getFrame(aic::CapturedFrame &frame)
 
 void aic::VideoInput::run()
 {
-    VideoType videoType = VideoType::NONE;
```
Comment (agriic) [inline src/VideoInput.cpp]: Always initialize

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,342 @@
+# CMAKE generated file: DO NOT EDIT!
```
Comment (agriic) [inline Makefile]: Delete generated makefile

File: src/VideoInput.hpp [hunk 3]
```diff
@@ -33,7 +33,7 @@ class VideoInput
     bool useCaptureCamera = false;
     int captureCamera = 0;
 
-    std::atomic<bool> stopped;
+    std::atomic<bool> stopped{};
```
Comment (agriic) [inline src/VideoInput.hpp]: Stopped is still initialised to random value - better change it back.

**Review by agriic — CHANGES_REQUESTED** [review]

**Comment by danirukun** [comment]
Pushed changes

**Review by agriic — APPROVED** [review]

**Pull request merged**